gravatar/gravatar-tests.ts

```diff
@@ -0,0 +1,14 @@
+/// <reference path="gravatar.d.ts" />
+
+import gravatar from 'gravatar';
```

Comment:
Please change to `import gravatar = require('gravatar');` or The gravatar type definition support `default export` of ES6 format.